Applications/PaintBrush/LineTool.cpp

```diff
@@ -3,6 +3,19 @@
 #include <LibGUI/GAction.h>
 #include <LibGUI/GMenu.h>
 #include <LibGUI/GPainter.h>
+#include <LibM/math.h>
+
+Point constrain_line_angle(const Point& start_pos, const Point& end_pos, float angle_rad)
```

Comment:
`angle_rad` is `angle_increment_in_radians`, right?

Reply:
> `angle_rad` is `angle_increment_in_radians`, right?

Yes, I've changed it to `angle_increment`, because it is actually pretty obvious that every angle there is in radians.